src/components/panel/panel.js

```diff
@@ -649,6 +652,8 @@ function MdPanelService($rootElement, $rootScope, $injector) {
 MdPanelService.prototype.create = function(opt_config) {
   var configSettings = opt_config || {};
 
+  angular.copy(this._defaultConfigOptions, this._config);
```

Comment:
nit: What about using extend for both? As written, it harder to remember that one of these copies left to right and the other right to left.